internal/manifests/console/consoleexternalloglink.go

```diff
@@ -89,3 +91,24 @@ func MutateConsoleExternalLogLink(current, desired *consolev1.ConsoleExternalLog
 	current.Spec.HrefTemplate = desired.Spec.HrefTemplate
 	current.Spec.Text = desired.Spec.Text
 }
+
+func DeleteConsoleExternalLogLink(ctx context.Context, c client.Client) error {
+	current := &consolev1.ConsoleExternalLogLink{}
+	key := client.ObjectKey{Name: ConsoleExternalLogLinkName}
+	err := c.Get(ctx, key, current)
+	if err != nil {
+		return kverrors.Wrap(err, "failed to get consoleexternalloglink",
+			"name", ConsoleExternalLogLinkName,
+		)
+	}
```

Comment:
Why do we need to get this before deleting? Why not simply construct a stub for:
```go
current := &consolev1.ConsoleExternalLogLink{}
```
by setting only ObjectMeta Name and Namespace?

Reply:
you're right! And even better, we don't need a namespace since this is a cluster wide resource